packages/ui/client/components/dashboard/ErrorEntry.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+  import { isPrimitive } from 'vitest/utils'
```

Comment:
Please don't import from `vitest/` in UI package. Only relative paths are allowed there. This particular function can just be inlined